server/src/main/java/org/opensearch/Version.java

```diff
@@ -90,7 +90,8 @@ public class Version implements Comparable<Version>, ToXContentFragment {
     public static final Version V_1_3_7 = new Version(1030799, org.apache.lucene.util.Version.LUCENE_8_10_1);
     public static final Version V_1_3_8 = new Version(1030899, org.apache.lucene.util.Version.LUCENE_8_10_1);
     public static final Version V_1_3_9 = new Version(1030999, org.apache.lucene.util.Version.LUCENE_8_10_1);
-    public static final Version CURRENT = V_1_3_9;
+    public static final Version V_1_3_10 = new Version(10301099, org.apache.lucene.util.Version.LUCENE_8_10_1);
```

Comment:
@owaiskazi19 looks like the version number from auto increment workflow has problems. 
Which is why the tests are assuming LegacyES Version. Give this a shot.
```suggestion
    public static final Version V_1_3_10 = new Version(1031099, org.apache.lucene.util.Version.LUCENE_8_10_1);
```

Reply:
Thanks for directing to the right version id. Pushed the chnage